pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -7399,10 +7399,7 @@ spec:
   serviceAccountName: default
 `)
 
-	cms := []*corev1.ConfigMap{withEnabledAlphaAPIFields(newFeatureFlagsConfigMap())}
-
 	d := test.Data{
-		ConfigMaps:   cms,
```

Comment:
👍

Reply:
Actually, let's open a separate issue for this, since we're not testing `v1` in the reconciler tests anywhere at the moment - that seems like a separate requirement for `v1` rather than specific to remote resolution beta, and will involve a lot of tests.

Reply:
ok, sounds good